apps/cookbook/src/app/component-overview/component-overview.component.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
Same as the other test file :) Consider adding more tests, than just the autogenerated ones :)

Reply:
As we figured out we don't run test in cookbook 😄